### PR TITLE
[sc-29601] Assign author for documents

### PR DESCRIPTION
**Pull request by usefulalgorithm**
### 🤔 Why?

Now that we can impersonate as a user when creating knowledge cards, in the CLI we want to actually use it do create data documents on behalf of some existing user.

### 🤓 What?

- Added option for `aesop documents create` and `aesop glossary import` to assign author to the document.
- Added unit test

I did not add a lot of tests here because writing a test is somewhat finicky now: instead of calling the functions, we have to manually call the CLI in a test runner and process stdout. In a follow up PR I will refactor it so that the commands are directly callable, which would make test writing a lot easier.

### 🧪 Tested?

- Added unit test.
- Tested against andy.dev.metaphor.io.

### CODE REVIEW

File: aesop/commands/common/entity_id.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import re
+
+from aesop.graphql.generated.enums import EntityType
+
+
+def is_entity_id(
+    x: str,
+    entity_type: EntityType,
+) -> bool:
+    return re.match("^" + entity_type.value + "~[\\dA-F]{32}$", x) is not None
```
Comment (mars-lan) [inline aesop/commands/common/entity_id.py:10]:
```suggestion
    return re.match("^" + entity_type.value.upper() + "~[\\dA-F]{32}$", x) is not None
```

**Review by mars-lan — COMMENTED** [review]
Otherwise LGTM

File: aesop/commands/common/entity_id.py (hunk 1 above)
Comment (usefulalgorithm) [reply aesop/commands/common/entity_id.py:10]: It is already in uppercase

**Review by alyiwang — APPROVED** [review]
LGTM

**Pull request merged**